content/zh/docs/setup/additional-setup/sidecar-injection/index.md

```diff
@@ -203,17 +212,47 @@ spec:
 通常，可以设置 Pod 中的任何字段。但是必须注意某些字段：
 
 * Kubernetes 要求在注入运行之前配置 `image`。虽然可以您可以设置一个指定的 Image 来覆盖默认的 `image` 配置，但建议将 `image` 设置为 `auto`，可使 Sidecar 注入自动选择要使用的 Image。
+
 * `Pod` 中一些字段取决于相关设置。例如，CPU 请求必须小于 CPU 限制。如果两个字段没有一起配置， Pod 可能会无法启动。
 
-另外，某些字段可通过在 Pod 上的 [Annotations](/zh/docs/reference/config/annotations/) 配置，但是不建议使用上述方法进行自定义设置。
+另外，某些字段可通过在 Pod 上的[注解](/zh/docs/reference/config/annotations/)进行配置，但是不建议使用上述方法进行自定义设置。必须特别注意某些注解：
+
+* 如果设置了 `sidecar.istio.io/proxyCPU`，则务必显式设置 `sidecar.istio.io/proxyCPULimit`。否则该 Sidecar 的 `cpu` 限制将被设置为 unlimited。
+
+* 如果设置了 `sidecar.istio.io/proxyMemory`，则务必显式设置 `sidecar.istio.io/proxyMemoryLimit`。否则该 Sidecar 的 `memory` 限制将被设置为 unlimited。
+
+例如，参见以下不完整的资源注解配置和相应注入的资源设置：
 
-### 自定义模板[试验特性]{custom-templates-[experimental]}
+{{< text yaml >}}
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/proxyCPU: "200m"
+        sidecar.istio.io/proxyMemoryLimit: "5Gi"
+{{< /text >}}
+
+{{< text yaml >}}
+spec:
+  containers:
+  - name: istio-proxy
+    resources:
+      limits:
+        memory: 5Gi
+      requests:
+        cpu: 200m
+        memory: 5Gi
+      securityContext:
+        allowPrivilegeEscalation: false
+{{< /text >}}
+
+### 自定义模板（试验特性）{#custom-templates-experimental}
```

Comment:
```suggestion
### 自定义模板（试验特性）{#customizing-injection}
```

keep anchor with En doc

Reply:
Feel a little strange. The original heading is 

### Custom templates (experimental)

Reply:
sorry, it's my mistake, ignore this.